apps/portal-front/app/(embed)/layout.tsx

```diff
@@ -55,11 +56,16 @@ const RootLayout: FunctionComponent<RootLayoutProps> = async ({ children }) => {
     return (
       <RelayProvider>
         <PageLoader>
-          <ContentLayout>
-            <Card className="w-4/5 m-auto mt-l p-l h-auto bg-page-background">
-              {children}
-            </Card>
-          </ContentLayout>
+          <div className="flex flex-col items-center justify-center min-h-screen">
+            <div className="w-1/3">
+              <LogoXTMDark className="p-6" />
+              <ContentLayout>
+                <Card className="w-full p-xl bg-page-background">
```

Comment:
need to keep `w-full p-xl` ?

```suggestion
                <Card className="bg-page-background">
```

Reply:
The w-full no, removed (ok)
But the padding is needed otherwise everything sticks to the border 😞